code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -142,33 +147,31 @@
 					adjust_discipline(1, TRUE)
 
 			if(follow_mob) // We're being asked to stop following someone.
-				if(follow_mob == speaker)
+				if(can_command(speaker) == SLIME_COMMAND_FRIEND || follow_mob == speaker)
 					delayed_say("Yes... I'll stop...", speaker)
 					LoseFollow()
 				else
 					delayed_say("No... I'll keep following \the [follow_mob]...", speaker)
 
-		// Help request
```

Comment:
This was removed because we couldn't think of a way to make it work?

Reply:
There's no easy and robust way to determine what is 'threatening' to the person asking help.

Reply:
Ok